dev/com.ibm.ws.jsp/src/com/ibm/ws/jsp/translator/compiler/JDTCompiler.java

```diff
@@ -125,20 +128,20 @@ else if (jdkSourceLevel == 15) {
         }
         //PM04610 start
         else if (jdkSourceLevel == 16) {
-            compilerOptionsMap.put(CompilerOptions.OPTION_Source, CompilerOptions.VERSION_1_6);
+            compilerOptionsMap.put(CompilerOptions.OPTION_Source, CompilerOptions.VERSION_1_6); 
```

Comment:
Can you clean up the unnecessary white space after these compilerOptionsMap entries?

Reply:
The current formatting in the com.ibm.ws.jsp project is a mess is some places -- I created this PR to format the entire project: https://github.com/OpenLiberty/open-liberty/compare/integration...volosied:format-jsp?expand=1

Let me know what you think.

Reply:
I believe it used the rules declared in `.settings/org.eclipse.jdt.core.prefs`

Reply:
Yea we can do this as a follow on PR. We can also update all the settings in : https://github.com/OpenLiberty/open-liberty/issues/27172